api-lib/gql/__generated__/zeus/index.ts

```diff
@@ -22109,11 +22111,24 @@ const handleFetchResponse = (
 export const apiFetch =
   (options: fetchOptions) =>
   (query: string, variables: Record<string, any> = {}) => {
-    let fetchFunction = fetch;
```

Comment:
the changes below won't get stomped by re-generation?

Reply:
@levity the addition of the `--node` flag on the zeus command for server-side libgen is what generated this node-friendly code: 

https://github.com/coordinape/coordinape/blob/c688cb6452f38632f34d0e08fd4d3b78fbc3aa96/scripts/generate-zeus.sh#L47